packages/sui-studio/workbench/src/app.js

```diff
@@ -64,15 +65,30 @@ const importAll = requireContext => requireContext.keys().map(requireContext)
 
   const {raw} = params
   const ComponentToRender = raw ? Raw : Root
+
+  const routes = (
+    <Router>
```

Comment:
You don't need to wrap the `routes` with another <Router> that's in fact a Context Provider.

Reply:
I know, I tried to follow the same pattern as the tests, but for some reason it didn't work.

Until I wrapped everything up in a second Router always seemed like a blank page.